delfin/drivers/ibm/storwize_svc/alert_handler.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import time
+
+from oslo_log import log
+from delfin.common import constants
+from delfin import exception
+from delfin.drivers.ibm.storwize_svc import consts
+
+LOG = log.getLogger(__name__)
+
+
+class AlertHandler(object):
+    default_me_category = 'storage-subsystem'
```

Comment:
Constant should be named by CAPS_WITH_UNDERSCORE

Reply:
Class do not need